sprockets/mixins/correlation/mixins.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, *args, **kwargs):
         # one exists.  We also want to set it in the outgoing response
         # which the property setter does for us.
         maybe_future = super(HandlerMixin, self).prepare()
-        if concurrent.is_future(maybe_future):
+        if concurrent.is_future(maybe_future):  # pragma: nocover
```

Comment:
Don't ignore this as it's fairly critical functionality. A hole in coverage is preferable for now.

Reply:
Removed